docs/wxm-devices/wifi-m5-bundle/m5-connect-and-claim.mdx

```diff
@@ -165,7 +165,7 @@ Once the WiFi credentials have been successfully transferred to M5 Miner, the de
 
 
 <div class="explanation small-big">
-  <img src={useBaseUrl("img/wxm-devices/wifi-m5-bundle/serial-number.svg")} />
+  <img src={useBaseUrl("img/wxm-devices/wifi-m5-bundle/serial-number.jpg")} />
   <span>
```

Comment:
@epapangelis Please delete the old .svg file if it's not needed anymore in any other occasion.